src/cli/cli.cpp

```diff
@@ -606,6 +609,50 @@ void Interpreter::ProcessDomainName(uint8_t aArgsLength, char *aArgs[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_CONFIG_DUA_ENABLE
+void Interpreter::ProcessDua(uint8_t aArgsLength, char *aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(aArgsLength >= 1 && strcmp(aArgs[0], "iid") == 0, error = OT_ERROR_INVALID_COMMAND);
+
+    switch (aArgsLength)
+    {
+    case 1:
+    {
+        const otIp6InterfaceIdentifier *iid_ptr = otThreadGetFixedDuaInterfaceIdentifier(mInstance);
+
+        if (iid_ptr != NULL)
+        {
+            OutputBytes(iid_ptr->m8, sizeof(otIp6InterfaceIdentifier));
+            mServer->OutputFormat("\r\n");
+        }
```

Comment:
A blank line as below is what you are expecting?
```
> dua iid

Done
```

Reply:
I do prefer always output a line of iid so that we can use code like below in `node.py`:
```python
def get_dua_iid(self):
    self.send_command('dua iid')
    return self._expect_results(r'\S+')
```
Thoughts?

Reply:
If so, It may not be consistent with existing style, like `prefix`